### PR TITLE
DOCSP-5659 - Add Kubernetes Operator to the Tools page.

### PR DESCRIPTION
**Pull request by jdestefano-mongo**
Staged: https://docs-mongodborg-staging.corp.mongodb.com/landing/jdestefano/DOCSP-5659/tools/index.html

Must be merged with https://github.com/mongodb/docs-tools/pull/296 if we need the centered card.

### CODE REVIEW

**Comment by jdestefano-mongo** [comment]
Five cards is weird, I'm not sure if we want to center or left align the fifth.

File: src/html/tools.html [hunk 1]
```diff
@@ -64,7 +68,17 @@ <h3 class="card__title">MongoDB Compass</h3>
         </p>
       </a>
 
-      <a class="card" href="https://docs.mongodb.com/spark-connector/current/">
+      <a class="card" href="https://docs.mongodb.com/kubernetes-operator/stable/">
+        <span class="card__icon card__icon--document"></span>
+        <h3 class="card__title">MongoDB Kubernetes Operator</h3>
+        <p class="card__body">
+          Reference guide for the MongoDB Enterprise Kubernetes Operator. Learn
```
Comment (jamesbroadhead) [inline src/html/tools.html]: Not sure this first sentence is needed. 

Maybe something like: 
"Learn how to run MongoDB on Kubernetes using our Kubernetes Operator, and monitor and back them up with Cloud or Ops Manager"  

?

File: src/html/tools.html [hunk 2]
```diff
@@ -16,7 +16,11 @@
           </li>
           <li class="toc__item">
             <a class="toc__link" href="https://docs.mongodb.com/compass/current/">MongoDB Compass</a>
+          </li>              
+          <li class="toc__item">
+            <a class="toc__link" href="https://docs.mongodb.com/kubernetes-operator/stable/">MongoDB Enterprise Kubernetes Operator</a>
```
Comment (jamesbroadhead) [inline src/html/tools.html]: This doesnt quite fit -- maybe just "MongoDB Kubernetes Operator" 
or: "Enterprise Kubernetes Operator" 

either works for me.

File: src/html/tools.html (hunk 1 above)
Comment (jdestefano-mongo) [reply src/html/tools.html]: Going with you suggestion tweaked a little bit.

File: src/html/tools.html (hunk 2 above)
Comment (jdestefano-mongo) [reply src/html/tools.html]: Sure, I'll go with "MongoDB Kubernetes Operator" and change this on the main docs landing page.

**Pull request merged**